frappe/public/js/frappe/form/grid_row.js

```diff
@@ -298,6 +298,10 @@ export default class GridRow {
 				setTimeout(function() {
 					$(col).find('input[type="Text"]:first').focus();
 				}, 500);
+				
+				var evt_name = me.parent_df.fieldname + "_grid_row_after_click";
+				frappe.web_form.events.trigger(evt_name, me);
```

Comment:
You cannot add `web_form` events in form.js

Reply:
Could you elaborate more on why this is not possible?
In practice, it works nicely in enabling this feature of dynamic select elements in forms.

Reply:
Largely components and forms is supposed to be independent of web_forms, it's just a matter of separation. Of course it may work correctly for your site, but it does not align the conventions of frappe.